pkg/resource/events-logger-config/alloy/events-logger-config.alloy.yaml.template

```diff
@@ -47,6 +47,45 @@ alloy:
     replicas: 1
   crds:
     create: false
+  extraObjects:
+  - apiVersion: v1
```

Comment:
It needs to bé flagged behind tracingenabled right?

Reply:
Oh yeah you're right